gpfit/fit.py

```diff
@@ -202,5 +202,5 @@ def rfun(p):
         if K == 1:
             cstrt = MonomialEquality(mono2, "=", mono1)
 
-    return cstrt, rmsErr
```

Comment:
I'm embarrassed to say that we don't have any unit tests that actually enforce that the first output of `fit` is a constraint. But the tests do use the variable name `cstrt`, so the change to params makes the tests confusing:

```
./t_ex6_3.py:    cstrt, rms_error = fit(x, y, K, "ISMA")
./t_ex6_3.py:    cstrt, rms_error = fit(x, y, K, "SMA")
./t_ex6_3.py:    cstrt, rms_error = fit(x, y, K, "MA")
```